modules/compose.mjs

```diff
@@ -282,12 +293,12 @@ class ReplyWithHeader {
             }
 
             if (headers[hdrKey]) {
-                rwhHeaders += '<p style="margin:0cm;font-size:11.5pt"><span><b>' + lbl + '</b> ' + headers[hdrKey] + '</span></p>';
+                rwhHeaders += '<p style="margin:0"><span><b>' + lbl + '</b> ' + headers[hdrKey] + '</span></p>';
```

Comment:
@ygoe I can't recall why I added `font-size:11.5pt`, but I'm sure there is a reason.
I obtained this from an Outlook client. I'm okay with removing the font-size value (we can wait for user feedback later). But preserve the margin with units either `0cm` or `0px`, whichever you prefer. Typically, unit less works well on webpages than email client HTML rendering.

Reply:
That's the main point, this font-size is wrong and needs to go away. It sets the font to something else than the default size and it's also a size that cannot even be selected by the user. If this is new to you, consider this one of that feedback you got on other compatibility issues. And setting the size is also unnecessary because none of the other text has one. And that is precisely how to get the default font size: don't set any.

I have reverted the style optimisation here but left out the font-size. Without it, my PR is pointless.

Reply:
I appreciate your understanding. People use various email clients, and RWH users report that their emails are displayed incorrectly on the receiver side.
Even the old Thunderbird client (before TB started using Firefox as the base) used to have email rendering issues and differences.

As I mentioned before, we can remove the font-size and see if any users get back to me with an issue. Then I will figure out how to fix it.

Reply:
Okay. I was also looking at the actually generated HTML (save as draft and view source of that message) to inspect things. There's no font-size anywhere within the regular content or Thunderbird's own reply or forward headers. So I wonder how adding one would fix anything.